sevntu-checkstyle-idea-extension/pom.xml

```diff
@@ -16,6 +16,10 @@
 
   <name>Sevntu Checkstyle Idea extension</name>
 
+  <properties>
+    <java.version>11</java.version>
```

Comment:
With the deploy.sh fix applied, javadoc wouldn't generate a jar because it was empty. Hence me moving the file to the correct maven location.

Then javadoc would fail and then wouldn't compile because the project was not specified as Java 11.